src/backend/db/products.js

```diff
@@ -97,28 +96,27 @@ export const products = [
 		badge: 'Out of Stock',
 		image: civilization,
 		price: 2499,
+		discount: 50,
 		rating: 3,
-		wishlist: false,
 		categoryName: 'Strategy',
 	},
 	{
 		_id: uuid(),
 		title: 'Age of Empires 2',
 		badge: '',
 		image: AOE2,
-		price: 1099,
+		price: 899,
 		rating: 3.5,
-		wishlist: false,
 		categoryName: 'Strategy',
 	},
```

Comment:
Only when I add this product it is showing a NaN value, here it doesn't have a discount property, maybe this is the reason why it is showing NaN for the total price value.

Reply:
Thank you finding out the bug, I will fix it right away.